docs/content/basic/authenticating-oauth.md

```diff
@@ -4,256 +4,703 @@ lang: en
 slug: /concepts/authenticating-oauth
 ---
 
-To prepare your Slack app for distribution, you will need to enable Bolt OAuth and store installation information securely. Bolt supports OAuth and will handle the rest of the work; this includes setting up OAuth routes, state verification, and passing your app an installation object which you must store. 
+OAuth allows installation of your app to any workspace and is an important step
+in distributing your app. For each completed authentication process, you receive
+unique [access tokens and related information](#the-installation-object) that
+can be retrieved for incoming events and used to make scoped API requests.
```

Comment:
```suggestion
OAuth allows installation of your app to any workspace and is an important step in distributing your app. For each completed authentication process, you receive unique [access tokens and related information](#the-installation-object) that can be retrieved for incoming events and used to make scoped API requests.
```

docusaurus quirk: The unfurl will show the first _line_  so for the first paragraph best to keep it on one line to stop it only showing half a sentence.

Reply:
Shoot I will forget this often... But TIL! I'm wondering if this could be checked by a linter at some point? Like, the first non-frontmatter line can only be a single line? IDK. I **can** hope to remember this for now though!

I feel like it'd be nice to ask ya- are broken lines elsewhere something you're a fan of? I find it easier to edit, but please do not let that bias an answer 🙏